pkg/apis/kops/componentconfig.go

```diff
@@ -759,6 +759,28 @@ type CloudConfiguration struct {
 	Openstack *OpenstackConfiguration `json:"openstack,omitempty"`
 }
 
+// ComponentConfig contains the configuration for kops components.
+type ComponentConfig struct {
+	// NodeTerminationHandlerConfig determines the cluster autoscaler configuration.
+	NodeTerminationHandler *NodeTerminationHandlerConfig `json:"nodeTerminationHandler,omitempty"`
+}
+
+// NodeTerminationHandlerConfig determines the cluster autoscaler configuration.
+type NodeTerminationHandlerConfig struct {
+	// Enabled enables the node termination handler.
+	// Default: false
+	Enabled *bool `json:"enabled,omitempty"`
+	// EnableSpotInterruptionDraining makes node termination handler drain nodes when spot interruption termination notice is received.
+	// Default: true
+	EnableSpotInterruptionDraining *bool `json:"enableSpotInterruptionDraining,omitempty"`
```

Comment:
I'm not quite sure why someone would want to turn this off.

Reply:
Someone asked for this on slack. They wanted just the cordon.